src/OrbitGl/StringManager.cpp

```diff
@@ -12,6 +12,11 @@
 
 namespace orbit_gl {
 
+StringManager* StringManager::GetInstance() {
+  static auto instance = std::make_unique<StringManager>();
```

Comment:
Why a unique_ptr? A `static StringManager instance;` should work just fine.

Reply:
This is now obsolete